test_scripts/Defects/7_1/1619_AlertManeuer_empty_SoftButtons_missing_Info.lua

```diff
@@ -0,0 +1,77 @@
+--------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1619
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with valid AlertManeuver response (empty parameter "info")
+-- to the request RPC AlertManeuer with empty softbuttons
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session are started
+-- 3) App is registered
+-- 4) PTU has been performed
+-- 5) App is activated
+--
+-- Steps:
+-- 1) Send AlertManeuver mobile RPC from app with parameter softButtons = []
+-- 2) HMI responds with SUCCESS to Navigation.AlertManeuver request
+--
+-- SDL does:
+-- 1) respond with 'AlertManeuver', { success = true, resultCode = "SUCCESS" }
+--    and missing parameter "info" to App
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+local json = require('modules/json')
```

Comment:
Please notice there is a possibility to use `json` through `actions`. E.g. `common.json.EMPTY_ARRAY`